security/answers.md

```diff
@@ -0,0 +1,6 @@
+# My new security setup
+
+    I have to admit that I have never used a password manager in my life. I would simply write them 
+down on a notepad and have them saved locally on a browser. While it works for the most part, one of the downsides is that over the years, the amount of passwords I have to maintain made it more difficult to maintain notepad documents and sometimes can get lost. So, I decided to give password managers a try. I don't think I feel comfortable using one that store things into a cloud because of concerns over databases potentially being hacked but the least I could do is to start with a local one. This is the main reason why I went with KeePass. 
```

Comment:
I was the same way for a long time with passwords, looks like you've found a pretty good solution!